Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,87 @@
+# Glossary
+
```

Comment:
A few overall thoughts:
- Let's define Rook terms that are found in the Rook docs. It would be good to have a definition for each CRD, and link each one to its CRD doc.
- Let's not redefine the K8s and Ceph terms. Definitions like CSI, StorageClass, PVC, and PV, seems like we should just let them find those in the K8s glossary. To make it searchable on the glossary page, maybe we just mention the names of some of the common K8s or Ceph terms, then link to those pages. A few of the really common K8s and Ceph terms may make sense to include in the Rook glossary, but very few.
- Let's define things in terms of Rook, and not copy definitions from other glossaries.

Thanks for putting this together. It's difficult to see in advance how the docs should look!